### DISCUSSION TITLE
Ds18ReScan and Ds18RetryRead unknown in 13.3 ?

### DISCUSSION THREAD

**Discussion by MaxVonEvil** [arendst/Tasmota#20419] [Q&A]
Hi, only discovered these after the 13.3 upgrade although it looks like they were implemented in 13.2. I have the occasional dropoff, however on 13.3 the commands seem not to exist (unknown command) 

Except for the release notes, I see no mention of these commands. Were they dropped from release or do I have to fire up my TasmoCompiler and do a custom firmware?

**Comment by sfromis** [comment]
Those commands become available if you have a custom build with
`#define DS18x20_USE_ID_ALIAS`
